holmes/core/conversations_worker/realtime_manager.py

```
@@ -0,0 +1,378 @@
+        def _on_subscribe_cb(status, err=None) -> None:
+            logging.info(
+                "RealtimeManager subscribe status=%s err=%s",
+                status,
+                err,
+            )
+            subscribed.set()
+            # Trigger a claim to cover any missed events during subscription setup
+            try:
+                self.on_new_pending()
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)